### ISSUE TITLE
crash relating to fabric containers on master branch code

### ISSUE THREAD

**Issue by Nathan22211** [LemmaEOF/Kiln#1]
https://gist.github.com/Nathan22211/126158dd5b0b7fee828c204a40f03b8b this is from trying to run the source code in Jetbrains, but I'd imagine something similar will occur with a modImplementation or a modCompileOnly of the mod via Jitpack

EDIT: seems to only affect the source posted to github, dunno why, but something's up

**Comment by LemmaEOF** [comment]
lemmaeFear

**Comment by LemmaEOF** [comment]
I might have forgotten to commit or smth? I have no idea I'll check my other machine tomorrow

**Comment by Nathan22211** [comment]
Yeah, also noticed jijing from curse and then compiling makes the mod it's compiled in crash, but I tested that more than a month ago and am not sure if it's still the case
